src/visualization/client/Turi Create Visualization/src/user_interface/src/index.js

```diff
@@ -2,13 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 import TcPlot from './elements/Plot/Chart/index.js';
-import TcSummary from './elements/Plot/Summary/index.js';
 import TcTable from './elements/Explore/Table/index.js';
 import TCEvaluation from './elements/Explore/Evaluation/index.js';
 
 import messageFormat from './format/message';
 
-import { load, Root } from 'protobufjs';
+import Root from 'protobufjs';
```

Comment:
I think this should be `import { Root } from 'protobufjs'` -- the import semantics are different between named and default exports.